### PR TITLE
CHAT-187 - Add a Guide to Debugging. Stop tracking docs.

### PR DESCRIPTION
**Pull request by ianjennings**
Adds a guide to docs describing how to debug ChatEngine.

screencapture-localhost-8080-tutorial-debugging-html-1519939956309

### CODE REVIEW

**Comment by ianjennings** [comment]
@crgbaumgart Here is the relevant file rendered in markdown:
https://github.com/pubnub/chat-engine/blob/0afa02f060710d200400f90cd5efd6d6d92994b9/guide/concepts/events/debugging.md

You can also run ```gulp docs_dev``` to serve the docs locally from this branch.

File: guide/concepts/events/debugging.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+This is effectively the same as writing:
+
+```js
+ChatEngine.onAny((event, payload) => {
+   console.log(event, payload);
+});
+```
+
+Beware that this is very verbose and should not be enabled in production as it
```
Comment (crgbaumgart) [inline guide/concepts/events/debugging.md]: Just a preference thing, could we partial bold this sentence? To stress the importance?

**Comment by ianjennings** [comment]
@crgbaumgart made requested changes and added an extra note about verbosity to the config docs 👍

**Comment by crgbaumgart** [comment]
@ianjennings I just saw the change, looks great! Ty.

**Comment by ianjennings** [comment]
@crgbaumgart can merge but needs another approval

**Pull request merged**